validator/client/beacon-api/duties.go

```diff
@@ -121,9 +122,24 @@ func (c *beaconApiValidatorClient) dutiesForEpoch(
 			if err != nil {
 				return errors.Wrapf(err, "failed to parse attester committee index `%s`", attesterDuty.CommitteeIndex)
 			}
-			attesterDutiesMapping[primitives.ValidatorIndex(validatorIndex)] = committeeIndexSlotPair{
-				slot:           primitives.Slot(slot),
-				committeeIndex: primitives.CommitteeIndex(committeeIndex),
+			committeeLength, err := strconv.ParseUint(attesterDuty.CommitteeLength, 10, 64)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse attester committee length `%s`", attesterDuty.CommitteeLength)
+			}
+			validatorCommitteeIndex, err := strconv.ParseUint(attesterDuty.ValidatorCommitteeIndex, 10, 64)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse attester validator committee index `%s`", attesterDuty.ValidatorCommitteeIndex)
+			}
+			committeesAtSlot, err := strconv.ParseUint(attesterDuty.CommitteesAtSlot, 10, 64)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse attester committees at slot `%s`", attesterDuty.CommitteesAtSlot)
+			}
+			attesterDutiesMapping[primitives.ValidatorIndex(validatorIndex)] = attDuty{
+				slot:                    primitives.Slot(slot),
+				committeeIndex:          primitives.CommitteeIndex(committeeIndex),
+				committeeLength:         committeeLength,
+				validatorCommitteeIndex: validatorCommitteeIndex,
+				committeesAtSlot:        committeesAtSlot,
```

Comment:
this value is not set on gRPC but can be resolved in a subsequent PR, we are not currently using it for gRPC